tests/Valkey.Glide.IntegrationTests/ConnectionMultiplexerReadFromMappingTests.cs

```diff
@@ -0,0 +1,446 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+using static Valkey.Glide.ConnectionConfiguration;
+
+namespace Valkey.Glide.IntegrationTests;
+
+/// <summary>
+/// Integration tests for ConnectionMultiplexer ReadFrom mapping functionality.
+/// These tests verify that ReadFrom configuration flows correctly from ConfigurationOptions
+/// through to the ClientConfigurationBuilder and ConnectionConfig levels.
+/// </summary>
+[Collection(typeof(ConnectionMultiplexerReadFromMappingTests))]
+[CollectionDefinition(DisableParallelization = true)]
+public class ConnectionMultiplexerReadFromMappingTests(TestConfiguration config)
+{
+    public TestConfiguration Config { get; } = config;
+
+    [Fact]
+    public async Task ConfigurationOptions_ReadFromPrimary_MapsToStandaloneClientConfigurationBuilder()
+    {
+        // Arrange
+        var configOptions = new ConfigurationOptions
+        {
+            ReadFrom = new ReadFrom(ReadFromStrategy.Primary)
+        };
+        configOptions.EndPoints.Add(TestConfiguration.STANDALONE_HOSTS[0].host, TestConfiguration.STANDALONE_HOSTS[0].port);
+        configOptions.Ssl = TestConfiguration.TLS;
+
+        // Act
+        var connectionMultiplexer = await ConnectionMultiplexer.ConnectAsync(configOptions);
```

Comment:
```suggestion
        using var connectionMultiplexer = await ConnectionMultiplexer.ConnectAsync(configOptions);
```
And you don't need dispose then
Applicable to all